rust/private/repository_utils.bzl

```diff
@@ -671,7 +671,8 @@ def load_arbitrary_tool(ctx, tool_name, tool_subdirectories, version, iso_date,
             urls.append(new_url)
 
     tool_path = produce_tool_path(tool_name, target_triple, version)
-    archive_path = tool_path + _get_tool_extension(ctx)
+
```

Comment:
Is the new line needed?

Reply:
My thought was that the archive path is no longer calculated using the tool path created in the previous line, so it sort of made sense to separate it visually.

I've moved this code out into a new function though.